packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.kt

```diff
@@ -104,8 +101,7 @@ open class FileDownloader(context: Context, private val client: OkHttpClient) {
     val responseHeaderData = ResponseHeaderData(
       protocolVersionRaw = responseHeaders["expo-protocol-version"],
       manifestFiltersRaw = responseHeaders["expo-manifest-filters"],
-      serverDefinedHeadersRaw = responseHeaders["expo-server-defined-headers"],
-      manifestSignature = responseHeaders["expo-manifest-signature"]
```

Comment:
Confirmed at this is considered a legacy manifest feature server-side as well. We still need to keep the server-side code for old clients for awhile (maybe a year after Expo Go for SDK 51 has been released?) but this change to the client looks good.